plugins/techdocs/src/reader/transformers/html/hooks/iframes.ts

```diff
@@ -37,6 +37,22 @@ const isSafe = (node: Element, hosts: string[]) => {
   }
 };
 
+/**
+ * Checks whether a iframe is safe or not.
+ * @param node - is an iframe element.
+ * @param hosts - list of allowed hosts.
+ * @returns true when iframe is included in hosts.
+ */
+export const isHost = (node: Element, check: string) => {
+  const src = node.getAttribute('src') || '';
+  try {
+    const { host } = new URL(src);
+    return host.includes(check);
+  } catch {
+    return false;
+  }
+};
```

Comment:
Hey, just want to make sure I'm understanding this right.  Is the main difference between this function and `isSafe` that this one works on a single host, while `isSafe` uses an array?  

Also, is it super important that we're checking for a substring here, or could we potentially reuse `isSafe` by passing the host in as a single-item array (like `isSafe(node, [host])`)?  Maybe we could even update `isSafe` to handle both arrays and single strings?